README.md

```
@@ -12,19 +12,41 @@
 
-### License
-This project is licensed under the AGPL-3.0 license. See the [LICENSE](LICENSE) file for details.
+*More tools coming soon...*
+
+
+# Self hosting
+At the moment, only frontend app can be self-hosted on your own server. Therefore, it is not possible to sync your data between devices using self-hosted app. To self-host and access the offline-only frontend app using your own web URL - Please follow the instructions below.
+
+1. Clone/Fork the [client](https://gitlab.com/21nOrg/client) repository
+2. Set Environment Variables. Use any of `memotron`, `pointron`, or `nucleus` for `VITE_PRODUCT`
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)